libs/datasets/can_raw_location_page_urls.py

```diff
@@ -0,0 +1,67 @@
+import structlog
```

Comment:
WDYT about creating a top-level `scripts` directory like covid-data-public had?  That way it's clearer that this is a standalone script that's not part of the main data pipeline code?  Putting it under `libs/datasets` seems a bit confusing to me.  Open to other ideas though.

Reply:
yeah that sgtm 🙏 , done in e5357366e03317c325c11282651a25b5905dcbd5